### ISSUE TITLE
autocorrect from empty_else cop will raise an exception if there are two elsifs

### ISSUE THREAD

**Issue by alexanderadam** [bbatsov/rubocop#3569]
## Expected behavior

`rubocop` should not raise an exception

## Actual behavior

it raises an exception

## Steps to reproduce the problem

try to autocorrect this:

```ruby
if 'hi'
  puts 'bbatsov'
elsif 'this'
  puts 'will'
elsif 'break'
  puts 'rubocop'
else
  nil
end

```

## RuboCop version
```
0.43.0 (using Parser 2.3.1.4, running on ruby 2.1.1 x86_64-linux
```

**Comment by rrosenblum** [comment]
I am surprised that this hasn't come up earlier. The issue appears to be caused by the cop checking an `elsif` expression instead of the `else` expression. A similar issue most likely exists in `Stlye/MissingElse`. I have an idea on how to fix this.

**Comment by rrosenblum** [comment]
It looks like this was a duplicate of #3382. @lumeet fixed this in #3571.
